artifactory/cli.go

```diff
@@ -680,6 +680,13 @@ func getBuildToolFlags() []cli.Flag {
 	}
 }
 
+func getSkipLoginFlag() cli.Flag {
```

Comment:
Both docker push and docker pull are using the same flags such as:
getBuildToolFlags, getServerFlags and getSkipLoginFlag. The only difference is in the docker push command that has also the getThreadsFlag. Perhaps we should create a getDockerFlags that will include the getBuildToolFlags, getServerFlags and getSkipLoginFlag and for docker push also add the threads flags:

```
func getDockerFlags() []cli.Flag {
	var flags []cli.Flag
	flags = append(flags, getBuildToolFlags()...)
	flags = append(flags, getServerFlags()...)
	flags = append(flags, getSkipLoginFlag())
	return flags
}

func getDockerPushFlags() []cli.Flag {
	var flags []cli.Flag
	flags = append(flags, getDockerFlags()...)
	flags = append(flags, getThreadsFlag())
	return flags
}
```